src/utils/navigation.ts

```diff
@@ -189,6 +189,11 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.BENEFICIAL_OWNER_STATEMENTS_URL,
     nextPage: [config.BENEFICIAL_OWNER_STATEMENTS_URL, config.BENEFICIAL_OWNER_TYPE_URL]
   },
+  [config.BENEFICIAL_OWNER_DELETE_WARNING_WITH_PARAMS_URL]: {
+    currentPage: config.BENEFICIAL_OWNER_DELETE_WARNING_PAGE,
+    previousPage: () => config.BENEFICIAL_OWNER_STATEMENTS_URL,
+    nextPage: [config.BENEFICIAL_OWNER_STATEMENTS_URL, config.BENEFICIAL_OWNER_TYPE_URL]
```

Comment:
I think this needs to be BENEFICIAL_OWNER_STATEMENTS_WITH_PARAMS_URL. Worth making sure you have the latest code from the feature branch and running it in Tilt.

Reply:
ok, we can look at its usage in another task.